src/main/java/io/confluent/connect/jdbc/JdbcUtils.java

```diff
@@ -51,7 +51,7 @@
    * The default only includes standard, user-defined tables.
    */
   public static final Set<String> DEFAULT_TABLE_TYPES = Collections.unmodifiableSet(
-      new HashSet<>(Arrays.asList("TABLE"))
+      new HashSet<>(Arrays.asList("TABLE", "VIEW"))
```

Comment:
This is potentially a compatibility issue since existing users will start seeing views show up in there. We actually already have a similar PR: https://github.com/confluentinc/kafka-connect-jdbc/pull/37 I kind of like the combination of the two patches -- this one has an actual test with Derby instead of just mocking `JdbcUtils` and the `validateNonNullable` tweak, but the other one includes the relevant config, maintains compatibility, and generalizes.

If I get the other one merged, would rebasing against that make sense for you? (The merge is trivial, I just want to make sure the combo would work for your use case.)